examples/worlds/graded_buoyancy.sdf

```diff
@@ -0,0 +1,222 @@
+<?xml version="1.0" ?>
+<!--
+  Ignition Gazebo buoyancy plugin demo. This world demonstrates a graded
+  buoyancy world consisting of some basic shapes which should eventually float
+  up.
+-->
```

Comment:
It would be interesting to add a semi-transparent plane to show where the layers change. That can be just a static model with a visual and no collision.

It would also be interesting to add:

* A shape that sinks slowly
* A third layer :smirk: